run.sh

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env bash
 
 __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
-$__scriptpath/init-tools.sh
-if [ $? -ne 0 ]; then
-    exit 1
-fi
+. $__scriptpath/init-tools.sh
```

Comment:
I think this is a reasonable approach but can you please add a comment here saying why we are souring it so it doesn't get undone later?